cmd/openshift-install/create.go

```diff
@@ -385,9 +386,13 @@ func waitForConsole(ctx context.Context, config *rest.Config, directory string)
 	}
 
 	consoleRouteTimeout := 10 * time.Minute
-	logrus.Infof("Waiting up to %v for the openshift-console route to be created...", consoleRouteTimeout)
 	consoleRouteContext, cancel := context.WithTimeout(ctx, consoleRouteTimeout)
 	defer cancel()
+	if oneShot {
+		logrus.Infof("Checking for the %s route...", consoleNamespace)
```

Comment:
`/s/consoleNamespace/consoleRouteName`

Reply:
> `/s/consoleNamespace/consoleRouteName`

My follow up is waiting for the `downloads` route too; this choice will be a better fit then.  Even without follow-up work, I think "the main console-namespace route" isn't that bad vs. naming the route without the namespace.  If we end up sticking with a single route, I'll probably write both the namespace and name.